workflow-runtime/src/commonMain/kotlin/com/squareup/workflow1/SimpleLoggingWorkflowInterceptor.kt

```diff
@@ -113,7 +115,7 @@ public open class SimpleLoggingWorkflowInterceptor : WorkflowInterceptor {
     println(text)
   }
 
-  protected open fun logError(text: String): Unit = System.err.println(text)
+  protected open fun logError(text: String): Unit = println("ERROR: $text")
```

Comment:
Not sure how important this is, but it should be possible to implement multiplatform printing to stderr.